src/core/agent/agent.types.ts

```diff
@@ -120,6 +120,7 @@ interface KeriaNotification {
   multisigId?: string;
   connectionId: string;
   read: boolean;
+  groupReplied: boolean;
```

Comment:
@Sotatek-DukeVu @sdisalvo-crd Please see description on this, and why. When displaying `/ipex/apply` (requests) notifications in the UI, it should display as it currently is _unless_ `groupReplied=true;` and we are NOT the leader of the multisig.

The `multisigId` combined with Redux should allow you to quickly check this.

We have changed `connectionsCache` to be a `Record<string, ConnectionShortDetails>` instead of `ConnectionShortDetails[]`. Can we do this for identifiers too for fast look up?

Reply:
Thanks @iFergal. 

```We have changed connectionsCache to be a Record<string, ConnectionShortDetails> instead of ConnectionShortDetails[]. Can we do this for identifiers too for fast look up?```

I will do it in other task.

Reply:
@iFergal 
`The multisigId combined with Redux should allow you to quickly check this.`
I tested on local and notification data after the text changed not contain `multisigId`. Can you or @jimcase  @Sotatek-BaoHoanga @Sotatek-TungNguyen2a help me check it?

Reply:
notification value:
```
{
        id: '0AA5z0UUdAFJOJfO7hsvo8BC',
        createdAt: '2024-12-17T08:32:49.550Z',
        a: {
          r: '/exn/ipex/apply',
          d: 'EAipIu_jC5ErFrhL0qTKgMt2rhD1PSkcRsWFSEZOknFF',
          m: ''
        },
        connectionId: 'EBklXnWIITD78SBkbtpH3qYxpLAlwmLH0udSmOOlOkkz',
        read: false,
        groupReplied: true
 }

Reply:
@Sotatek-DukeVu You should get the `multisigId` field from https://github.com/cardano-foundation/cf-identity-wallet/blob/8bf3329c9775fc65737cb0c0a56f1a0b620bf375/src/core/agent/services/keriaNotificationService.ts#L850

Reply:
Is this ok now @Sotatek-DukeVu or still an issue?

Reply:
Yes. @Sotatek-TungNguyen2a helped me fix this issue in my PR: https://github.com/cardano-foundation/cf-identity-wallet/pull/877.